modules/040-node-manager/templates/capi-controller-manager/static-cluster.yaml

```diff
@@ -0,0 +1,39 @@
+{{- $prefix := "static" }}
+{{- $provider := "static" }}
+---
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: {{ $prefix }}
+  namespace: d8-cloud-instance-manager
+  {{- include "helm_lib_module_labels" (list . (dict "app" "capi-controller-manager")) | nindent 2 }}
+  annotations:
+    cluster.x-k8s.io/paused: "true"
+# A finalizer has been added to prevent cascading removing of all capi resources after deleting the cluster resource.
+  finalizers:
+    - deckhouse.io/capi-controller-manager
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks:
+        - {{ .Values.global.clusterConfiguration.podSubnetCIDR }}
+    services:
+      cidrBlocks:
+        - {{ .Values.global.clusterConfiguration.serviceSubnetCIDR }}
+    serviceDomain: {{ .Values.global.clusterConfiguration.clusterDomain }}
+#  infrastructureRef:
```

Comment:
dead code

Reply:
no, this is not the dead code, we'll uncomment them when we deploy capi-static-provider. Capi cluster resource should have reference to underlying infrastructure cluster resource to work.